frontend/src/app/components/experiments/experiment-template-list-item/experiment-template-list-item.component.html

```diff
@@ -2,15 +2,23 @@
     <mat-card-header [routerLink]="['/experiments/templates', template.id]">
         <mat-card-subtitle>{{template.id}}</mat-card-subtitle>
         <mat-card-title>
-            <h3>{{template.name}}</h3>
+            
+            <span class="template-name">
+                <h3>{{template.name}}</h3>
+                <mat-icon *ngIf="template.is_public" matTooltip="Public template">public</mat-icon>
+                <mat-icon *ngIf="!template.is_public" matTooltip="Private template">public_off</mat-icon>
+            </span>
         </mat-card-title>
     </mat-card-header>
     <mat-card-content>
-        <div class="state" *ngIf="template.state != 'FINISHED'">
-            <ng-container *ngIf="!template.approved">
+        <div class="state" *ngIf="template.is_archived">
+            <span class="crashed"><mat-icon matTooltip="New experiments cannot use this template">delete</mat-icon>Archived</span>
+        </div>
+        <div class="state" *ngIf="!template.is_archived && template.state != 'FINISHED'">
+            <ng-container *ngIf="!template.is_approved">
                 <span class="crashed"><mat-icon matTooltip="Experiment templates created by users are manually reviewed and then approved for use">search</mat-icon>Waiting for approval</span>
```

Comment:
This is actually the same message as in `experiment-template-detail`. Can it be somehow unified?
"Experiment templates are manually reviewed before being approved for use"